### PR TITLE
feat(GA): add new resources endpoint and endpoint group

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

add new resources endpoint and endpoint group

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
add new resources endpoint and endpoint group
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/ga' TESTARGS='-run TestAccEndpointGroup_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/ga -v -run TestAccEndpointGroup_basic -timeout 360m -parallel 4
=== RUN   TestAccEndpointGroup_basic
=== PAUSE TestAccEndpointGroup_basic
=== CONT  TestAccEndpointGroup_basic
--- PASS: TestAccEndpointGroup_basic (94.69s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/ga        94.828s

make testacc TEST='./huaweicloud/services/acceptance/ga' TESTARGS='-run TestAccEndpoint_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/ga -v -run TestAccEndpoint_basic -timeout 360m -parallel 4
=== RUN   TestAccEndpoint_basic
=== PAUSE TestAccEndpoint_basic
=== CONT  TestAccEndpoint_basic
--- PASS: TestAccEndpoint_basic (117.61s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/ga        117.785s
```

### CODE REVIEW

File: huaweicloud/services/ga/resource_huaweicloud_ga_endpoint.go [hunk 1]
```diff
@@ -0,0 +1,541 @@
...
+			},
+			"health_state": {
+				Type:     schema.TypeString,
+				Computed: true,
+				Description: `|-
+                    Specifies the health check result of the endpoint. The value can be one of the following:
+                      - **INITIAL**: Initial
+                      - **HEALTHY**: Healthy
+                      - **UNHEALTHY**: Unhealthy
+                      - **NO_MONITOR**: Not monitored`,
```
Comment (Lance52259) [inline huaweicloud/services/ga/resource_huaweicloud_ga_endpoint.go]:
```suggestion
				Description: `Specifies the health check result of the endpoint.`,
```

File: huaweicloud/services/ga/resource_huaweicloud_ga_endpoint.go [hunk 2]
```diff
@@ -0,0 +1,541 @@
...
+			},
+			"status": {
+				Type:     schema.TypeString,
+				Computed: true,
+				Description: `|-
+                    Specifies the provisioning status. The value can be one of the following:
+                      - **ACTIVE**: The resource is running.
+                      - **PENDING**: The status is to be determined.
+                      - **ERROR**: Failed to create the resource.
+                      - **DELETING**: The resource is being deleted.`,
```
Comment (Lance52259) [inline huaweicloud/services/ga/resource_huaweicloud_ga_endpoint.go]: ditto

File: huaweicloud/services/ga/resource_huaweicloud_ga_endpoint_group.go [hunk 3]
```diff
@@ -0,0 +1,571 @@
...
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				Description: `|-
+                    Specifies the endpoint group name. The name can contain 1 to 64 characters.
+                    Only letters, digits, and hyphens (-) are allowed.`,
```
Comment (Lance52259) [inline huaweicloud/services/ga/resource_huaweicloud_ga_endpoint_group.go]: ditto

File: huaweicloud/services/ga/resource_huaweicloud_ga_endpoint.go (hunk 1 above)
Comment (Jason-Zhang9309) [reply huaweicloud/services/ga/resource_huaweicloud_ga_endpoint.go]: fixed

File: huaweicloud/services/ga/resource_huaweicloud_ga_endpoint.go (hunk 2 above)
Comment (Jason-Zhang9309) [reply huaweicloud/services/ga/resource_huaweicloud_ga_endpoint.go]: fixed

File: huaweicloud/services/ga/resource_huaweicloud_ga_endpoint_group.go (hunk 3 above)
Comment (Jason-Zhang9309) [reply huaweicloud/services/ga/resource_huaweicloud_ga_endpoint_group.go]: fixed

**Pull request merged**